code/modules/food_and_drink/popsicles.dm

```diff
@@ -37,7 +37,7 @@
 			user.put_in_hand_or_drop(P)
 			if(prob(8))
 				P.melt(user)
-			qdel(src)
+			dispose(src)
```

Comment:
Why?

Reply:
You know, I think I misunderstood which of these two I should be calling.